src/node/plugin.ts

```diff
@@ -23,38 +27,58 @@ require("module")._load = function (request: string, parent: object, isMain: boo
   return originalLoad.apply(this, [request.replace(/^code-server/, path.resolve(__dirname, "../..")), parent, isMain])
 }
 
+/**
+ * Load a plugin and run its activation function.
+ */
 const loadPlugin = async (pluginPath: string, httpServer: HttpServer, args: Args): Promise<void> => {
   try {
     const plugin: Plugin = require(pluginPath)
     plugin.activate(httpServer, args)
-    logger.debug("Loaded plugin", field("name", path.basename(pluginPath)))
+
+    logger.debug(
+      "Loaded plugin",
+      field("name", path.basename(pluginPath)),
```

Comment:
Maybe this should be from `package.json` as well?

Reply:
And then a separate field for the path.

Reply:
Ooo yeah much better.